app/main.py

```diff
@@ -62,6 +63,11 @@
     has been specifically advised of the possibility of such damages.'''
 
 
+class ModelEnum(str, Enum):
```

Comment:
Probably better if we have a dedicated file for storing this kind of class, thought?

Reply:
I've actually moved this into models/fetch.py in the next PR already - ok if we leave it here for now?

Reply:
Sure